handbook/engineering/security/index.md

```diff
@@ -39,12 +39,11 @@ These goals represent our targeted work for 70% of our time. The remaining 30% i
 
 **Milestones:**
 
-1. We need a better way to store and access [our own secrets](https://docs.google.com/document/d/1HzO7szEm-h4fqlQOnVbcJdpDmfQiM7Rb-Tz4CMEYl-Q).
```

Comment:
I don't think we need this for GA. I think it would be great to have, but we need more headcount for this.